### PR TITLE
Use a mask instead of an img for "Show image" eye

### PR DESCRIPTION
**Pull request by turt2live**
See https://github.com/matrix-org/matrix-react-sdk/pull/3492#pullrequestreview-296223646

### CODE REVIEW

File: src/components/views/messages/MImageBody.js [hunk 1]
```diff
@@ -496,7 +496,7 @@ export class HiddenImagePlaceholder extends React.PureComponent {
         return (
             <div className={className}>
                 <div className='mx_HiddenImagePlaceholder_button'>
```
Comment (t3chguy) [inline src/components/views/messages/MImageBody.js]: Can this become an AccessibleButton whilst you're here

Comment (turt2live) [reply src/components/views/messages/MImageBody.js]: It's not a real button.

**Comment by turt2live** [comment]
@t3chguy ping

**Comment by t3chguy** [comment]
oh whoops, I dismissed my issue with it so presumed it'd be back in the queue, but alas it is not

**Comment by turt2live** [comment]
ah, I missed the resolution happening - thanks!

**Pull request merged**